edx_django_utils/user/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""
+All user relevant code will live here.
```

Comment:
Hmmm... The fact that we'd need a new README.rst for `/user`, and it would need to be for any random user related utilities, makes me wondering if this is the right location for this code. I'm wondering whether this belongs with other auth code in `edx-drf-extensions`, or perhaps a new repo? I can discuss more with arch-bom, but maybe you can also help make the case for this repo if you continue to believe this is where it belongs.

Reply:
Nimisha reminded me that edx-drf-extensions could be called edx-api-provider, which clarifies that it is not the right place. She mentioned user-utils as a possibility. I’m not sure who’s input is required to make a final decision.

Reply:
@robrap I recommended this repository to @NIXKnight because it is called "edx-django-utils", as it seems to contain a collection of utilities for Django services. I am struggling to see why monitoring/admin/db/plugin/caching utilities belong here, but user management utilities don't. I am also surprised that we'd want to create new repository for two user management commands given the challenges that repository proliferation has brought us, although maybe I'm overestimating the impact of those challenges versus the impact of bloat within repositories.

In any case, I'd expect that the final decision is up to the Arch-BOM team as owners of this repository.

Reply:
@kdmccormick: 
1. This repo was a reasonable recommendation, and still may end up being the final location. Note that when setting up this repo, we knew it would be a careful balance to not making this a dumping ground for everything, but that makes it unclear what should and should not land here.
2. The repo that Nimisha mentioned already exists, https://github.com/edx/user-util (I got the name wrong), but it is currently just used for retirement and doesn't have the django setup. It is an idea to be considered.
3. If this code did stay here, would it make more sense to have the `user` utilities, and should the management commands move here and not be a general area for any management commands, but instead be a part of supporting the user utilities?

Reply:
@estute As tech contact for the squad owning user-util, do you have an opinion on whether or not that would be an appropriate home for this?

Reply:
Thanks for clarifying @robrap . Knowing now that the `user-util` repo does exist, your suggestion makes a lot more sense to me. I'll wait for @estute 's thoughts on this code living there.

> If this code did stay here, would it make more sense to have the user utilities, and should the management commands move here and not be a general area for any management commands, but instead be a part of supporting the user utilities?

Are you asking whether the user management commands should go within the `user/` sub-package? If so, then yes, I agree the user commands should be nested in the same package as the user utilities.

Reply:
@robrap @kdmccormick just to be clear, we have to move only the code from the `user` directory to https://github.com/edx/user-util right?

Reply:
I haven't had a chance to go over this yet, but it's on my to-do list for early next week.

Reply:
My thought was that everything in this PR should be consolidated under the “user” directory introduced, including the management commands. This would be in place of having a global management command folder.

If we decide to move to a separate repo, the entire app would move, with all relevant changes. I don’t think this introduced any non-user related changes?

Reply:
Ah, thanks for the clarification Rob, I just saw this comment and hadn't understood that.  A


@NIXKnight Sounds like Rob is suggesting we consider closing this PR in favor of copying all of the logic to a user-utils PR if @estute and DE are ok with that.  Given that that work may take longer and I don't see a strong reason not to have it in this repo I have a slight preference to keeping it in this repo, but we should make sure to understand the decision and work with the repo owners to make that choice.  If needed we can ask the managers of Arch and DE to decide since they own the two repos.

@robrap Do you know if there any specific upsides or downsides to having this logic in `edx-django-utils` and is there a reason we have multiple utils repos / process for determining which code should live in which repo.  I think we scoped this ticket for `edx-django-utls` because @kdmccormick or/and I didn't know about the `user-utils` repo, so it could be moved, but since I don't know much about the two repos I am hesitant to do all the work of moving it.

Reply:
@adzuci: I was proposing an alternative to be considered. I think it may come with some pain. Let's discuss elsewhere with the right people, and summarize the decision here once it is made.